frontend/javascripts/oxalis/view/jobs/train_ai_model.tsx

```diff
@@ -425,19 +468,43 @@ function AnnotationsCsvInput({
           name: annotation.dataSetName,
         });
 
-        const volumeTracings: ServerVolumeTracing[] = await Promise.all(
+        const volumeServerTracings: ServerVolumeTracing[] = await Promise.all(
           annotation.annotationLayers
-            .filter((layer) => layer.typ === "Skeleton")
+            .filter((layer) => layer.typ === "Volume")
             .map(
               (layer) =>
                 getTracingForAnnotationType(annotation, layer) as Promise<ServerVolumeTracing>,
             ),
         );
+        // TODO: make bboxs a member again
```

Comment:
TODO

Reply:
ups, forgot to remove this. I already implemented it this way.